module/coriolis-roll.js

```diff
@@ -413,6 +456,150 @@ function getTooltipData(results) {
+    case "infiltration":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillInfiltration");
+    case "manipulation":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillManipulation");
+    case "meleecombat":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillMeleeCombat");
+    case "observation":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillObservation");
+    case "rangedcombat":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillRangedCombat");
+    case "survival":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillSurvival");
+    case "command":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillCommand");
+    case "culture":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillCulture");
+    case "datadjinn":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillDataDjinn");
+    case "medicurgy":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillMedicurgy");
+    case "mysticpowers":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillMysticPowers");
+    case "pilot":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillPilot");
+    case "science":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillScience");
+    case "technology":
+      return SkillKey = game.i18n.localize("YZECORIOLIS.SkillTechnology");
+  }
+  return SkillKey;
+}
+
+function getRollModifier(rollData) {
+  return `${rollData.modifier}`;
+}
+
+function getRollIsAuto(rollData) {
+  return `${rollData.weapon_isAutomatic}`;
+}
+
+function getRollIsAutoActive(rollData) {
+  return `${rollData.additionalData?.automaticFire}`;
+}
+
+function getRollAutoIgnoOnes(rollData) {
+  return `${rollData.additionalData?.numberOfIgnoredOnes}`;
+}
+
+function getRollIsExplosive(rollData) {
+  return `${rollData.weapon_isExplosive}`;
+}
+
+function getRollBonus(rollData) {
+  return `${rollData.bonus}`;
+}
+
+function getRollBlastPower(rollData) {
+  return `${rollData.weapon_blastPower}`;
+}
+
+function getRollBlastRadius(rollData) {
+  let radius = "";
+  switch (rollData.weapon_blastRadius) {
+    case "close":
+      return radius = game.i18n.localize("YZECORIOLIS.CloseRange");
+    case "short":
+      return radius = game.i18n.localize("YZECORIOLIS.ShortRange");
+    case "long":
+      return radius = game.i18n.localize("YZECORIOLIS.LongRange");
+    case "extreme":
+      return radius = game.i18n.localize("YZECORIOLIS.ExtremeRange");
+  }
+  return radius;
+}
+
+function getRollCrit(rollData) {
+  return `${rollData.weapon_crit}`;
+}
+
+function getRollCritText(rollData) {
+  return `${rollData.weapon_critText}`;
+}
+
+function getRollDmg(rollData) {
+  return `${rollData.weapon_damage}`;
+}
+
+function getRollDmgText(rollData) {
+  return `${rollData.weapon_damageText}`;
+}
+
+function getRollRange(rollData) {
+  let range = "";
+  switch (rollData.weapon_range) {
```

Comment:
Same comment as above.

Reply:
Yes, I wasn't sure how that is reusable. But I tryed it now till it works. (easy if you understand it >__<)